methods/guest/Cargo.toml

```diff
@@ -17,5 +17,11 @@ tree_hash = "0.9"
 tree_hash_derive = "0.9"
 
 [patch.crates-io]
-ethereum_hashing = { git = "https://github.com/ReamLabs/ethereum_hashing" } # Add ethereum_hashing::Sha256Context that's failing tree_hash crate used by ream
+#ethereum_hashing = { git = "https://github.com/ReamLabs/ethereum_hashing" } # Add ethereum_hashing::Sha256Context that's failing tree_hash crate used by ream
 sha2 = { git = "https://github.com/risc0/RustCrypto-hashes", tag = "sha2-v0.10.8-risczero.0" }
+# With poseidon2 hash
+ethereum_hashing = { git = "https://github.com/SuccinctPaul/ethereum_hashing", rev = "692ad3d" }
+
+
+[patch."https://github.com/ReamLabs/ethereum_hashing"]
+ethereum_hashing = { git = "https://github.com/SuccinctPaul/ethereum_hashing", rev = "692ad3d" }
```

Comment:
Can't we already be on `ethereum_hashing = { git = "https://github.com/ReamLabs/ethereum_hashing", branch = "poseidon2" }`?